R/parseTimeseries.R

```diff
@@ -18,6 +18,10 @@ parseTimeseries <- function(file, delim, with.units = FALSE){
   
   config = parseConfig(file, delim)
   
+  if(!is.na(config[['features']][1]) && config[['features']][1]=="") {
```

Comment:
Since config[['features']] can be a multipolygon, config[['features']][1] comes back NA in that case. When config[['features']][1]=="" that indicates that there is a preceding column with no name. The TIMESERIES column is already taken out so it can only be the threshold column. I had to do this up here because the blockData hasn't been created yet and I needed to cull out that empty string for creating of the dataOut dataframe.